syntax = "proto2";

// Annotations that can be applied to the output schema by analyzers or mappers.
package third_party.py.tensorflow_transform.annotations;

// Represents the bucket boundaries that were used to Bucketize a feature.
message BucketBoundaries {
  repeated float boundaries = 1;
}

// Represents metadata about the computed vocabulary.
message VocabularyMetadata {
  optional string file_name = 1;
  // The original size of the vocabulary, prior to any filtering (e.g.
  // filtering to top_k).
  optional int32 unfiltered_vocabulary_size = 2;
}
